lang/python/python-pycares/Makefile

```diff
@@ -2,11 +2,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pycares
-PKG_VERSION:=4.2.2
-PKG_RELEASE:=$(AUTORELEASE)
+PKG_VERSION:=4.3.0
+PKG_RELEASE:=2
```

Comment:
actaully it should be `1` ;)

Reply:
Yeah was confused about the suggestion provided, changed back into 1 through 85a4c59